witch_doctor.gemspec

```diff
@@ -18,14 +18,12 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["spec/**/*"]
 
-  s.add_dependency "rails", "~> 3.2"
+  s.add_development_dependency "rails", "~> 4.0.0"
```

Comment:
hmm technically the requirements are 
```
s.add_dependency "rails", ">= 3.2"
s.add_dependency "strong_params"
s.add_development_dependency "rails", "~> 4.0.0"
```

so I'm lost here too :)  generaly this isn't issue for our app but becouse this is a public gem this may be an issue let's keep it for now like this